.gitignore

```diff
@@ -103,6 +103,9 @@ solutions/swb-reference/integration-tests/config/*.yaml
 solutions/swb-reference/src/config/*.json
 solutions/swb-reference/.aws-sam/
 
+#exclude binary files
+**/*.rpm
+
```

Comment:
we probably shouldnt be ignoring all binaries, since there are some in SWB

Reply:
@SanketD92 ok with this ?

Reply:
We only want to ignore binaries in this folder: `solutions/swb-reference/src/environment-files/offline-packages/sagemaker/fuse-2.9.4` if there are any here. 

The ones in boto3 and fuse-2.9.4_AL2 are still needed.

Reply:
got it !